shell/initialize/plugins.js

```diff
@@ -1,34 +1,65 @@
 import '@shell/plugins/extend-router';
 import '@shell/plugins/formatters';
-import '@shell/plugins/global-formatters';
-import '@shell/plugins/i18n';
-import '@shell/plugins/vue-js-modal';
 import '@shell/plugins/js-yaml';
-import '@shell/plugins/portal-vue.js';
-import '@shell/plugins/resize';
-import '@shell/plugins/shortkey';
-import '@shell/plugins/tooltip';
-import '@shell/plugins/trim-whitespace';
-import '@shell/plugins/v-select';
+
+import globalFormatters from '@shell/plugins/global-formatters';
+import i18n from '@shell/plugins/i18n';
+import VModal from 'vue-js-modal';
+import PortalVue from 'portal-vue';
+import VueResize from '@shell/plugins/resize';
+import ShortKey from 'vue-shortkey';
+import VTooltip from 'v-tooltip';
+import TrimWhitespace from '@shell/plugins/trim-whitespace';
+import vSelect from 'vue-select';
+import intNumber from '@shell/plugins/int-number';
+import positiveIntNumber from '@shell/plugins/positive-int-number.js';
 
 import axios from '../utils/axios.js';
 import axiosShell from '@shell/plugins/axios';
 import backButton from '@shell/plugins/back-button';
 import codeMirror from '@shell/plugins/codemirror-loader';
 import cookieUniversalNuxt from '../utils/cookie-universal-nuxt.js';
-import * as intNumber from '@shell/plugins/int-number';
 import nuxtClientInit from '@shell/plugins/nuxt-client-init';
 import plugin from '@shell/plugins/plugin';
 import plugins from '@shell/core/plugins.js';
 import pluginsLoader from '../core/plugins-loader.js';
-import * as positiveIntNumber from '@shell/plugins/positive-int-number.js';
 import replaceAll from '@shell/plugins/replaceall';
 import steveCreateWorker from '@shell/plugins/steve-create-worker';
 import version from '@shell/plugins/version';
 import emberCookie from '@shell/plugins/ember-cookie';
 
-export async function installPlugins(app, inject) {
-  const pluginDefinitions = [cookieUniversalNuxt, axios, plugins, pluginsLoader, axiosShell, intNumber, positiveIntNumber, nuxtClientInit, replaceAll, backButton, plugin, codeMirror, version, steveCreateWorker, emberCookie];
+export async function installPlugins(app, inject, Vue) {
+  [
+    i18n,
+    globalFormatters,
+    VModal, PortalVue,
+    VueResize,
+    ShortKey,
```

Comment:
`Shortkey` is now missing the options `Vue.use(ShortKey, { prevent: ['input', 'textarea', 'select'] });`